### DISCUSSION TITLE
REST API

### DISCUSSION THREAD

**Discussion by tcitworld** [JulianPrieber/littlelink-custom#835] [Feature requests]
I would like to make the links manageable through an API.

A lot of organizations use LinkStack-like services to put temporary content like news articles on top of their links page, to avoid the Instagram limitation of not putting links in content description.

A few random examples:
- https://linkin.bio/wikimediafoundation
- https://linkin.bio/wikipedia/
- https://linktr.ee/GitLab

A 3rd-party small script could help getting elements from an RSS feed and always add the x latest elements on top, while removing older ones. Obviously if that use case is a feature that LinkStack want to have build-in, I'm interested as well.

**Comment by cfranz** [comment]
+1 from me ;)

I also have the need to automate some tasks via scripts. I have no expirience with laravel but with using google. I found this > https://api-platform.com/docs/laravel/

Is there a way to implement this to Linkstack?

**Comment by ajtatum** [comment]
I just installed LinkStack and really enjoy it. I was hoping to integrate it into a workflow in n8n or ActivePieces, but n8n integrates with Lnk,bio (as well as anything else with HTTP request).

Use case: new blog post published, create a new link on LinkStack
